### ISSUE TITLE
Broken Exclusions on Domains

### ISSUE THREAD

**Issue by jacoporusso-css** [ntop/ntopng#6905]
**Environment**:
* ntopng version/revision: v.5.4.220915]

**What happened**:
Exclusions about domains, implemented both via UI (starting directly from alert exclusion popup) and via ndpi protos file on ntopng and via ndpi file on nprobe have no effect.

Here's an example:
 We are flooded by "Binary Application Transfer" alerts on windows updates downloads

![ev_01](https://user-images.githubusercontent.com/108343272/193069700-6a23a60c-4cae-44ce-8c6e-bc3f5264bb9c.png)

So we tried to exclude it via UI

![ev_02](https://user-images.githubusercontent.com/108343272/193070789-d7a74cf4-255b-4a50-b50d-03e7179cd838.png)

And in fact it appears as excluded:

![ev_03](https://user-images.githubusercontent.com/108343272/193071438-4521a07f-b714-44b1-b5ee-965fdfb03699.PNG)

Anyway it had no effect, alert kept on generating.

So as stated [here](https://www.ntop.org/ndpi/howto-define-ndpi-risk-exceptions-for-networks-and-domains/) we created a new .protos file on ntopng instance, specifying the string:

host_risk_mask:".windowsupdate.com"=0xFFFFFFFFFFFFFFEF

and pointed at it via conf file (-p=/etc/ntopng/myfile.protos).
It had no effect, even restarting ntopng.

Finally we put the same string on nprobe instance (specifying --ndpi-custom-protos=/etc/nprobe/myfile.protos - anyway on the guide asks for PATH, not file, I believe it's a typo).
Alerts keep popping up, even after restarting nprobe.

If a pcap is needed to solve the issue, please let me know how to execute tcpdump to achieve this.

This issue is meant to be an extension of a previous one [#6886 ](https://github.com/ntop/ntopng/issues/6886), please feel free to close it.

Thank you

**Comment by jacoporusso-css** [comment]
Additional note, as stated [here](https://github.com/ntop/ntopng/issues/6886) this issue is showing only on numbered VLANs

**Comment by cardigliano** [comment]
This seems to be working now, with and without VLANs, with the full or a partial domain. 
Note that today I pushed a fix for the domains exceptions stored on redis that was creating unexpected behaviors.

As an example I tested exceptions with ftp.acc.umu.se and acc.umu.se as domains, with and without VLANS, using the below download to trigger the alert.
wget http://ftp.acc.umu.se/mirror/rockylinux.org/8.7/AppStream/x86_64/os/Packages/c/cups-client-2.2.6-50.el8.x86_64.rpm
